src/org/sparkboard/server/slack/core.clj

```diff
@@ -59,10 +60,17 @@
      (log/debug "[web-api] POST rsp:" rsp)
      rsp)))
 
-;; TODO "when a new member joins a project, add them to the linked channel"
-
 (comment
   (http-verb "/users.list")
-  
-  ;; TODO delete/archive channel, for testing and clean-up
+
   )
+
+(def channel-name
```

Comment:
Could use the `conversations.info` endpoint? (& then take `token` as a param). I am not sure that `web-api` will work for that though as i think it doesn’t support GET requests properly. I think something like `(http/get ... {:query {:token .. :channel ..}})` will work